### PR TITLE
[CP] Add citizen project landing page

### PR DESCRIPTION
**Pull request by illusionOfParadise**
The first commit was approved before. Concentrate on the second one.

### CODE REVIEW

File: src/Controller/EnMarche/CitizenProjectController.php [hunk 1]
```diff
@@ -269,11 +269,13 @@ public function unfollowAction(Request $request, CitizenProject $citizenProject)
...
      * @Method("GET")
      */
-    public function landingAction(): Response
+    public function landingAction(CitizenProjectManager $citizenProjectManager): Response
     {
         $city = $this->getUser() ? $this->getUser()->getCityName() : SearchParametersFilter::DEFAULT_CITY;
+        $citizenProjects = $citizenProjectManager->getAdherentCitizenProjectsAdministrator($this->getUser());
 
         return $this->render('citizen_project/landing.html.twig', [
+            'citizen_project' => $citizenProjects ? $citizenProjects[0] : null,
```
Comment (HeahDude) [inline src/Controller/EnMarche/CitizenProjectController.php]: `$citizenProjects[0] ?? null`

File: templates/citizen_project/_landing_results.html.twig [hunk 2]
```diff
@@ -1,6 +1,10 @@
 {% for citizen_project in results %}
     <li class="citizen-projects__landing__card">
-        <img src="{{ asset(citizen_project.imagePath) }}" alt="{{ citizen_project.name }}">
+        {% if citizen_project.hasImageUploaded %}
+            <div class="search__citizen_project__image">
+                <img src="{{ asset(citizen_project.getAssetImagePath) }}" alt="{{ citizen_project.name }}"/>
```
Comment (HeahDude) [inline templates/citizen_project/_landing_results.html.twig]: `citizen_project.assetImagePath`

File: templates/citizen_project/landing.html.twig [hunk 3]
```diff
@@ -105,17 +105,21 @@
...
             <div class="text--center">
-                <form id="start-creation-citizen-project" action="{{ path('app_adherent_create_citizen_project') }}" method="GET" class="b__nudge--top">
-                    <input type="text" name="name" placeholder="Le titre court et descriptif de mon projet">
-                    <button type="submit">
-                        Prochaine étape
-                        {{ include('components/arrow--right.html.twig') }}
-                    </button>
-                </form>
+                {% if citizen_project %}
+                    <a href="{{ path('app_citizen_project_show', {slug: citizen_project.slug}) }}">Voir mon projet {{ include('components/arrow--right.html.twig') }}</a>
```
Comment (HeahDude) [inline templates/citizen_project/landing.html.twig]: Spaces inside curly braces.

File: tests/Controller/EnMarche/CitizenProjectControllerTest.php [hunk 4]
```diff
@@ -496,9 +496,8 @@ public function testCitizenProjectLandingPageResultAuthenticate()
 
         $thumb3 = $crawler->filter('.citizen-projects__landing__card')->eq(2);
 
-        // This test fails only in CircleCI, need investigation
-//        $this->assertSame('Le projet citoyen à Marseille', trim($thumb3->filter('h3')->text()));
```
Comment (HeahDude) [inline tests/Controller/EnMarche/CitizenProjectControllerTest.php]: Is the comment still needed?

Comment (illusionOfParadise) [reply tests/Controller/EnMarche/CitizenProjectControllerTest.php]: No, the data was changed, there is no more this CP in test result.

Comment (HeahDude) [reply tests/Controller/EnMarche/CitizenProjectControllerTest.php]: So please remove it.

**Pull request merged**